README.md

```diff
@@ -53,14 +53,36 @@ This [example project][example] shows it in action.
 ## Configuration
 
 Firstly, setup [gazelle][gazelle] and [rules_haskell][rules_haskell].
-Then import `gazelle_haskell_modules`.
+Then import `gazelle_cabal` and `gazelle_haskell_modules`.
 
 ```python
+####################
+# Gazelle_cabal
+####################
+
+http_archive(
+    name = "io_tweag_gazelle_cabal",
+    strip_prefix = "gazelle_cabal-240ad1af8d0d0ca5c3fd3ff7afcdb2f0fe0dbbe2",
+    urls = ["https://github.com/tweag/gazelle_cabal/archive/240ad1af8d0d0ca5c3fd3ff7afcdb2f0fe0dbbe2.zip"],
+)
```

Comment:
It is a unsatisfying that we now have to bring `gazelle_cabal` even if the user does not intend to use it.

Reply:
Agree